src/Configuration/Context.php

```diff
@@ -14,6 +14,8 @@ public function getIpAddress(): ?string;
 
     public function getSessionId(): ?string;
 
+    public function getEnvironment(): ?string;
```

Comment:
This and `setEnvironment()` are a BC break which I would like to avoid, add them as class phpdoc methods for now.

Reply:
I'll do it